ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16-periodics.yaml

```diff
@@ -21941,18 +21743,7 @@ periodics:
     ci.openshift.io/generator: prowgen
     job-release: "4.16"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-gcp-ipi-f9-longduration-mco-critical
-  reporter_config:
-    slack:
-      channel: '#forum-ocp-mco-qe'
-      job_states_to_report:
-      - failure
-      - error
-      - success
-      report_template: '{{if eq .Status.State "success"}} :rainbow: Job *{{.Spec.Job}}*
-        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> :rainbow: {{else}}
-        :volcano: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
-        logs> :volcano: {{end}}'
+  name: periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-gcp-ipi-f9-longduration-mco-g1
```

Comment:
@Amoghrd mco jobs shouldn't have changed, do you need a rebase?

Reply:
I think that's a job resort, so github think there are changes, but actually there are no changes for mco jobs